samples/features/azure-arc/configuration/Patch-ComputeMachineDisableFeatures

```diff
@@ -0,0 +1,130 @@
+param (
+[string]$subscriptionId,
+[string]$resourceGroupName,
+[bool]$whatIf = $false
+)
+
+if ([string]::IsNullOrEmpty($subscriptionId)) {
+    $subscriptionId = Read-Host -Prompt "Please enter the subscription ID"
+}
+
+if ([string]::IsNullOrEmpty($resourceGroupName)) {
+    $resourceGroupName = Read-Host -Prompt "Please enter the resource group name"
+}
+
+if ([string]::IsNullOrEmpty($whatIf)) {
+    $dryRun = Read-Host -Prompt "Would you like to run this as a dry run? (yes/no)"
+    if ($dryRun -eq "yes") {
+            $whatIf = $true
+        } else {
+            $whatIf = $false
+        }
+}
+
+if ([string]::IsNullOrEmpty($resourceGroupName) -or [string]::IsNullOrEmpty($subscriptionId)) {
+    Write-Host "Either the resource group or subscription ID is not specified. Please provide both to proceed."
+    exit
+} else {
+    Write-Host "Performing actions on ResourceGroup: $($resourceGroupName) in Subscription:$($subscriptionId)"
+}
+
+$query = @"
+Resources 
+| where type == 'microsoft.azurearcdata/sqlserverinstances' 
```

Comment:
I think this ARG query initially i added to find SqlInstances, and from the instances, basically get the HybridComputeResource Id but at that time it was supposed to be a single script for both instance level and host level settings change. 
Now since it is only host level, can it be changed to query just hybrid compute machine ( from Sasha's script)

Reply:
I left it like this so we only change machines with SQL Server Instances, because we don't have to patch if they don't have our extension.